src/main/java/com/epam/repairstuidiospring/controller/OrderController.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.repairstuidiospring.controller;
+
+import com.epam.repairstuidiospring.controller.dto.OrderDto;
+import com.epam.repairstuidiospring.controller.dto.UserDto;
+import com.epam.repairstuidiospring.service.OrderService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@Slf4j
+@RequiredArgsConstructor
+public class OrderController {
+    private final OrderService orderService;
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(value = "/orders")
```

Comment:
I would recommend to keep one style of url, for this case keep the singular, like "/order" and please use @RequestMapping("/order") instead of duplicating the same url in each method